plugins/module_utils/network/sonic/argspec/port_breakout/port_breakout.py

```diff
@@ -43,9 +25,10 @@ def __init__(self, **kwargs):
             'options': {
                 'mode': {
                     'choices': ['1x10G', '1x25G', '1x40G', '1x50G', '1x100G',
-                                '1x200G', '1x400G', '2x10G', '2x25G', '2x40G',
-                                '2x50G', '2x100G', '2x200G', '4x10G', '4x25G',
-                                '4x50G', '4x100G', '8x10G', '8x25G', '8x50G'],
+                                '1x200G', '1x400G', '1x800G', '2x10G', '2x25G',
+                                '2x40G', '2x50G', '2x100G', '2x200G', '2x400G',
+                                '4x10G', '4x25G', '4x50G', '4x100G', '4x200G',
+                                '8x10G', '8x25G', '8x50G', '8x100G'],
```

Comment:
This is fewer modes than shown in the "module" file and fewer than were previously supported.

Even though four new modes were added here, the total is one less than before, which means that 5 modes got deleted.

(We had 20 before and now there are 19.)

The CLI shows 24 supported modes (excluding the one shown twice in the CLI "help"). The updated "module" file also shows 24.

CLI:  (1x100G is shown twice)

`sonic(config)# interface breakout port 1/1 mode
  <1x800G/2x400G/1x400G/4x200G/1x200G/2x200G/1x200G/8x100G/4x100G/2x100G/1x100G/8x50G/4x50G/2x50G/1x50G/2x40G/1x40G/8x25G/4x25G/2x25G/1x25G/8x10G/4x10G/2x10G/1x10G>`

Missing from the proposed argspec update: 1x10G, 1x25G, 1x40G, 1x50G, 1x100G